### PR TITLE
Adjustements to Swiss GMPEs for application in the Earthquake Risk Model of Switzerland

### PR DESCRIPTION
**Pull request by thpap**
* Adjusted the Swiss-specific implementations of the GMPEs used
    in the Swiss national seismic hazard (SUIhaz15) and risk (ERM-CH23) models.
    The new implementation returns all of total, intra- and inter-event sigmas,
    rather than just the total one.  
  * Extended the ModifiableGMPE class by adding an 
    apply_swiss_amplification_sa method that is used in ERM-CH23 to
    apply site specific adjustments for site effects and intra-event
    uncertainty for the two spectral accelerations used, SA(0.3s) and SA(0.6s). 
  * Added ch_ampl03, ch_ampl06, ch_phis2s03, ch_phis2s06,
    ch_phisss03, ch_phis2s06 site parameters to the site.py file.
    These parameters are required by the apply_swiss_amplification_sa method.

@danciul

### CODE REVIEW

File: openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py [hunk 1]
```diff
@@ -85,6 +85,27 @@ def apply_swiss_amplification(ctx, imt, me, si, ta, ph):
     me[:] += ctx.amplfactor
 
 
+def apply_swiss_amplification_sa(ctx, imt, me, si, ta, ph):
+    """
+    Adjust Swiss GMPEs to add amplification and correct intra-event residuals
+    """
+
+    if imt == from_string('SA(0.3)'):
```
Comment (micheles) [inline openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py:93]: The right way to do this check is
```python
if imt.period == 0.3:
```

File: openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py [hunk 2]
```diff
@@ -85,6 +85,27 @@ def apply_swiss_amplification(ctx, imt, me, si, ta, ph):
...
+        phis2s = ctx.ch_phis2s03
+        phiss = ctx.ch_phiss03
+        me[:] += ctx.ch_ampl03
+    elif imt == from_string('SA(0.6)'):
```
Comment (micheles) [inline openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py:97]:
```python
if imt.period == 0.6:
```

File: openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py (hunk 1 above)
Comment (thpap) [reply openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py]: done

File: openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py (hunk 2 above)
Comment (thpap) [reply openquake/hazardlib/gsim/mgmpe/modifiable_gmpe.py]: done

File: debian/changelog.bak [hunk 3]
```diff
@@ -0,0 +1,5633 @@
+  [Athanasios Papadopoulos]
```
Comment (micheles) [inline debian/changelog.bak:1]: I guess the changelog.bak file was committed by accident. Please remove it.

**Comment by micheles** [comment]
Please resolve the conflicts

**Comment by thpap** [comment]
> Please resolve the conflicts

@micheles Done

**Comment by micheles** [comment]
@ptormene we a have a big issue: the tests for any contributor (i.e. people working on a forked engine) will always be red because CI is trying to download the requirements from a non-existing branch. In this case for instance the offending command is
```
python3 -m pip install --trusted-host wheelhouse.openquake.org --trusted-host raw.githubusercontent.com -r https://raw.githubusercontent.com/gem/oq-engine/swiss-gmpes/requirements-py310-linux64.txt
```

**Pull request merged**